src/main.ts

```diff
@@ -1,12 +1,15 @@
 import "reflect-metadata";
 import { Peer } from "./concretes/peer";
-import { PeerType } from "./enums";
+import { PeerType } from "./enums/peerType.enum";
 import { IPeer } from "./interfaces/peer";
+import { Message } from "./models";
 
-const peer: IPeer = new Peer(PeerType.INITIAL_PEER);
+const peer: IPeer = new Peer(PeerType.VALIDATOR);
```

Comment:
Keep this main as an INITIAL_PEER node in the release and don't keep debugging code in there.